### PR TITLE
SAM-2537 for some reason the Samigo permissions were in localization files

### PR DESCRIPTION
**Pull request by ottenhoff**
This modification moves the permissions straight into the code as if someone
actually localized these strings it would be very bad.

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/AuthorAssessmentListener.java [hunk 1]
```diff
@@ -86,7 +86,7 @@ public void processAction(ActionEvent ae) throws AbortProcessingException
     
     AuthorizationBean authzBean = (AuthorizationBean) ContextUtil.lookupBean("authorization");
     if (!authzBean.isUserAllowedToCreateAssessment()) {
-        String err=(String)ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.AuthorMessages", "denied_create_assessment_error");
+        String err=(String)ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.AuthorMessages", "denied_assessment.createAssessment_error");
```
Comment (buckett) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/AuthorAssessmentListener.java]: This doesn't look right, I think your regex matched too much.

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/samlite/NameListener.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ public void processAction(ActionEvent ae) {
 		// Permission check
 	    AuthorizationBean authzBean = (AuthorizationBean) ContextUtil.lookupBean("authorization");
 		if (!authzBean.isUserAllowedToCreateAssessment()) {
-			String err=(String)ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.AuthorMessages", "denied_create_assessment_error");
```
Comment (buckett) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/samlite/NameListener.java]: This doesn't look right, I think your regex matched too much.

**Comment by buckett** [comment]
:fish_cake: for cleaning this up.

**Comment by ottenhoff** [comment]
Closing in favor of #637 ... I had the wrong branch reference in this PR.  I also fixed the @buckett comments.

**Pull request closed without merging**